caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -804,20 +805,27 @@
        * Form validation succeeded.
        */
       var onFormValid = function (eventData, args) {
+        var preserveColumnState = false
         var prevColumns = []
         var prevDisplayedColumns = []
         var prevColumnSelects = {}
         var prevSortOptions = {}
+        var currentCollections = $('#Observation\\.collection').val()
```

Comment:
This only handles the CAOM-2 form.  See the `form.getConfiguration().getDownloadAccessKey()` as an example of a way of making the field name generic and configured per Form Configuration.